.travis.yml

```diff
@@ -1,17 +1,41 @@
+jdk: oraclejdk8
+
 language: scala
 
 scala:
   - 2.12.4
   - 2.11.12
 
-env:
-  global:
-    - AKKA_TEST_TIMEFACTOR=2.0
+sudo: required
+
+services:
+  - docker
+
+script:
+  - travis_retry sbt clean coverage test
 
-sudo: false
+after_success:
+  - sbt coverageReport coveralls
 
-script: travis_retry sbt clean coverage test
+jobs:
+  include:
+    - stage: integration
+      scala: 2.12.4
+      env: AKKA_TEST_TIMEFACTOR=1.8
+      before_install:
+        - docker pull eventstore/eventstore
+        - docker run -d --rm --name eventstore-node -p 2113:2113 -p 1113:1113 -e EVENTSTORE_STATS_PERIOD_SEC=1200 eventstore/eventstore
+        - sleep 10
```

Comment:
why do we need this? is this enough ?

Reply:
Does not need to be there, since ES has started before client has been compiled. Will remove later